Whiskey/Functions/New-WhiskeyContext.ps1

```diff
@@ -232,6 +234,74 @@ Whiskey also no longer automatically adds build metadata to your version number.
         }
     }
 
+    if( $config.ContainsKey('PublishBuildStatusTo') )
+    {
+        Write-Warning -Message ('{0}: The "PublishBuildStatusTo" property is deprecated and will be removed in Whiskey 1.0. Please migrate to using the "PublishBuildStatusToBitbucket" task in the built-in "OnBuildStart" and "OnBuildEnd" pipelines in your whiskey.yml. Move the "Uri" and "CredentialID" properties to the new "PublishBuildStatusToBitbucket" task and then remove the deprecated "PublishBuildStatusTo" property from your whiskey.yml file.
+
+If you have a "PublishBuildStatusTo" property that looks like this:
+
+    PublishBuildStatusTo:
+    - BitbucketServer:
+        Uri: BITBUCKET_SERVER_URI
+        CredentialID: CREDENTIAL_ID
+
+Migrate to the "PublishBuildStatusToBitbucket" task like this:
+
+    OnBuildStart:
+    - TaskDefaults:
+        PublishBuildStatusToBitbucket:
+            Uri: BITBUCKET_SERVER_URI
+            CredentialID: CREDENTIAL_ID
+
+    - PublishBuildStatusToBitbucket
+
+    Build:
+    # ... SNIP ...
+
+    OnBuildEnd:
+    - PublishBuildStatusToBitbucket
+
+        ' -f $ConfigurationPath)
+
+        $publishToBitbucketTask = & {

```

Comment:
Let's not build pipelines on the fly. Instead, just output the warning about the behavior being obsolete and continue to run the `PublishBuildStatusTo` stuff the way we currently run it.